File: SERVIDOR CENTRAL/lib/presentation/ucb/ucbview/ucb_get_attendance_control_list_view.go
package ucbview

import (
	gosocketio "github.com/graarh/golang-socketio"
	"gitlab.com/jorgechavezrnd/servidorseminariogo2/lib/presentation/ucb/ucbviewmodel"
)

// UCBGetAttendanceControlListView ...
type UCBGetAttendanceControlListView struct {
	SocketIOChannel *gosocketio.Channel
}

// SendAttendanceControlList ...
func (ucbGetAttendanceControlListView UCBGetAttendanceControlListView) SendAttendanceControlList(model ucbviewmodel.UCBGetAttendanceControlListViewModel) {
	ucbGetAttendanceControlListView.SocketIOChannel.Emit("get_attendance_control_list_response", model)
}

File: SERVIDOR SDR_SCR/lib/presentation/ucb/ucbview/ucb_get_attendance_control_list_view.py
import asyncio
import socketio
from lib.presentation.ucb.ucbviewmodel.ucb_get_attendance_control_list_view_model import UCBGetAttendanceControlListViewModel

class UCBGetAttendanceControlListView:
    def __init__(self, sio: socketio.AsyncServer, sid):
        self.__sio = sio
        self.__sid = sid

    async def sendStudentLists(self, model: UCBGetAttendanceControlListViewModel):
        await self.__sio.emit('get_attendance_control_list_response', model.toJSON(), self.__sid)
        await self.__sio.sleep(1)